From 3ff8bf686bdbec280110fddd757f6a26942e0c2a Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Tue, 25 Jun 2024 12:21:17 -0700
Subject: [PATCH] Docs/translations/ko_KR: link howto.rst with other language
 versions

The menu for documents of other available languages is created for
documents in same file hierarchy under the translations/ directory.
Because howto.rst of Korean translation is at the root of translations/
directory while that for English is under howto/ directory, the Korean
translation is not linked with other available language versions via the
menu.  Move the document under the same hierarchy to make it be linked
with other langauge versions.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/translations/ko_KR/index.rst               | 2 +-
 Documentation/translations/ko_KR/{ => process}/howto.rst | 0
 2 files changed, 1 insertion(+), 1 deletion(-)
 rename Documentation/translations/ko_KR/{ => process}/howto.rst (100%)

diff --git a/Documentation/translations/ko_KR/index.rst b/Documentation/translations/ko_KR/index.rst
index 4add6b2fe1f2..2d51f1481310 100644
--- a/Documentation/translations/ko_KR/index.rst
+++ b/Documentation/translations/ko_KR/index.rst
@@ -11,7 +11,7 @@
 .. toctree::
    :maxdepth: 1
 
-   howto
+   process/howto
 
 
 리눅스 커널 메모리 배리어
diff --git a/Documentation/translations/ko_KR/howto.rst b/Documentation/translations/ko_KR/process/howto.rst
similarity index 100%
rename from Documentation/translations/ko_KR/howto.rst
rename to Documentation/translations/ko_KR/process/howto.rst
-- 
2.39.2

